packages/mui-joy/src/Link/Link.test.tsx

```diff
@@ -61,7 +61,7 @@ describe('<Link />', () => {
       const handlers = events.reduce((result, n) => {
         result[n] = spy();
         return result;
-      }, {});
+      }, {} as { [key: string]: any });
```

Comment:
```suggestion
      }, {} as Record<string, SinonSpy>);
```
`SinonSpy` is exported from 'sinon', to import it in this module you would add `import { SinonSpy } from 'sinon';`